### ISSUE TITLE
What GPM-DPR versions are compatible

### ISSUE THREAD

**Issue by wxwolff** [dopplerchase/DRpy#19]
Great project. Thanks!  I was wondering what versions of the DPR data DRpy supports.  What about the latest V7?

**Comment by dopplerchase** [comment]
Hi wxwolff, 

A major refactor happened when version 7 came out. So the code only currently works for 2A.DPR files that are version 7. 

This was centered around the change to file formats to have 1 zFactorMeasured (where Ku and Ka are in the same variable now). 

Cheers, 
Randy

**Comment by wxwolff** [comment]
Thanks! We (GPM GV) use it regularly and I am very glad to see you use Xarray.

David B. Wolff, Code 612
GPM Deputy Project Scientist for GV
NASA Wallops Flight Facility

**Comment by dopplerchase** [comment]
David, 

Thanks for reaching out! Please let me know if there are any errors bugs. This project is a side project for me, so I don't use it a ton to find all the bugs. 

Also, if you find you need something added, we can maybe work on adding stuff in. I know at one point Joe Munchak was going to share his GMI code with me. But i know he has been busy at Tomorrow.io 

Cheers, 
Randy

**Comment by wxwolff** [comment]
Sounds good. It sure would be nice if we could get at least GMI/GPROF and 2BCMB in here as well. I can reach out to Joe to see if I can get his GMI code.

**Comment by dopplerchase** [comment]
Yeah, a flexible loader like pyart would be nice, where it can automatically know what file you are using and load things into xarray. 

If end up applying for the NASA new investigators proposal call next year, maybe I will include some of my time to enhance DRpy.
